apps/mull-ui/src/app/components/event-card/event-card.tsx

```diff
@@ -32,8 +32,10 @@ export const EventCard = ({ event, style = {} }: EventCardProps) => {
         alt="Event"
       />
       <div className="event-card-datetime" data-testid="event-card-datetime">
-        <div>{`${month} ${day}`}</div>
+        <div style={{ fontWeight: 'bolder' }}>{day} </div>
+        <div>{month}</div>
         <div>{time}</div>
+        <div>{merīdiem}</div>
```

Comment:
Must the meridiem be on a separate line? I don't think it looks nice. And if it's not placed on a separate line, then it would simplify the logic you added in `apps/mull-ui/src/utilities.ts`

Reply:
meridiem is no longer used